spec/controllers/events_controller_spec.rb

```diff
@@ -358,18 +358,30 @@
   end
 
   describe "GET League#schedule" do
+    let(:league) { league = League.create! valid_league_attributes
+                   league }
+
     it "should generate schedule if not existing" do
-      event = League.create! valid_league_attributes
-      event.add_participant(@user)
-      event.add_participant(@other_user)
-      event.generate_schedule
-      get :schedule, params: { id: event.to_param }, session: valid_session
-      expect(event.matches).not_to be_empty
+      league.add_participant(@user)
+      league.add_participant(@other_user)
+      league.generate_schedule
+      get :schedule, params: { id: league.to_param }, session: valid_session
+      expect(league.matches).not_to be_empty
+    end
+
+    it "should update schedule if it is not up-to-date" do
+      league.gamedays << FactoryBot.create(:gameday)
+
+      last_gameday = league.gamedays.last
+      last_gameday.starttime = Date.today.next_day
+      last_gameday.endtime = Date.today.next_day 2
```

Comment:
Again I'd prefer to use `Date.current +1.day` or `Date.current + 2.days` here.

Reply:
Added to the list.

Reply:
Is there a difference to `Date.tomorrow`?